src/lib/message.js

```diff
@@ -1,4 +1,4 @@
-export default function message({ from, text, chat }) {
+export default function messageFactory({ from, text, chat }) {
```

Comment:
Like :) Was thinking about this myself but didn't have the energy at the time ^^